php/php.editor/src/org/netbeans/modules/php/editor/csl/PHPFoldingProvider.java

```diff
@@ -31,7 +31,8 @@
  */
 @MimeRegistration(mimeType = FileUtils.PHP_MIME_TYPE, service = FoldTypeProvider.class, position = 1000)
 public class PHPFoldingProvider implements FoldTypeProvider {
-    private static final Collection<FoldType> TYPES = new ArrayList<>(8);
+
+    private static final Collection<FoldType> TYPES = new ArrayList<>(9);
```

Comment:
Perhaps `Arrays.asList()`?

Reply:
Just a nitpick

Reply:
I also think so :) I'll fix it next time. Thanks!